storehaus-mongodb/src/main/scala/com/twitter/storehaus/mongodb/MongoStore.scala

```diff
@@ -93,24 +96,24 @@ class MongoStore[K: MongoValue, V: MongoValue: Manifest] (
 
   protected def getValue(valueObject: MongoDBObject): Option[V] = {
```

Comment:
Sorry I haven't been able to work on that for a while, I've been a bit overwhelmed with other things.

I wanted your input regarding this:

```scala

  protected def getValue(valueObject: MongoDBObject): Try[V] =
    valueObject.getAs[V](valueName) match {
      case None =>
        if (valueObject.get(valueName).isEmpty) // here
        else Throw(
          new ClassCastException(s"Cannot convert ${valueObject.get(valueName)} to ${manifest[V]}"))
      case Some(value) =>
        if (getManifest.isInstance(value)) Return(value)
        else Throw(new ClassCastException(s"Cannot convert ${value.getClass} to ${manifest[V]}"))
    }
```